consensus/istanbul/backend/backend.go

```diff
@@ -508,23 +566,21 @@ func (sb *Backend) HasBadProposal(hash common.Hash) bool {
 	return sb.hasBadBlock(hash)
 }
 
-func (sb *Backend) AddValidatorPeer(enodeURL string) {
-	if sb.broadcaster != nil {
-		sb.broadcaster.AddValidatorPeer(enodeURL)
+func (sb *Backend) addSentry(node, externalNode *enode.Node) error {
+	if sb.sentryNode != nil {
+		return errSentryAlreadySet
 	}
-}
 
-func (sb *Backend) RemoveValidatorPeer(enodeURL string) {
-	if sb.broadcaster != nil {
-		sb.broadcaster.RemoveValidatorPeer(enodeURL)
-	}
+	sb.p2pserver.AddPeerLabel(node, "sentry")
```

Comment:
Should we add the sentry as a trusted peer? I know generally this shouldn't be a problem to exceed maxpeers for a proxied validator, but technically it does represent the importance of the sentry?

Reply:
Adding a static peer will not impose any limits in regards to the max number of peers, if it's an outgoing connection.  Adding it as a trusted peer will not impose any limits to *incoming* connections (e.g. if the sentry were to try to connect to the validator).  Right now, all the infrastructure and the doc will always instruct that the validator be the initiator of the connection (and the code assumes that as well).